soh/src/code/z_parameter.c

```diff
@@ -1508,7 +1508,7 @@ void Inventory_SwapAgeEquipment(void) {
             gSaveContext.equips.equipment = gSaveContext.childEquips.equipment;
             gSaveContext.equips.equipment &= 0xFFF0;
             gSaveContext.equips.equipment |= 0x0001;
-        } else if (gSaveContext.n64ddFlag && Randomizer_GetSettingValue(RSK_STARTING_AGE)) {
+        } else if (gSaveContext.n64ddFlag && Randomizer_GetSettingValue(RSK_STARTING_AGE) == RO_AGE_ADULT) {
```

Comment:
If I'm understanding correctly, we shouldn't notice any difference here as this RSK should never === RO_AGE_RANDOM in game correct?

Reply:
exactly! when parsing the spoiler it's either child or adult

https://github.com/HarbourMasters/Shipwright/blob/8d398f71305c94acc9c231d239279a9a45918fa3/soh/soh/Enhancements/randomizer/randomizer.cpp#L627-L633